style/checkstyle.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
also update path in submodule pom:

- https://github.com/apache/incubator-hugegraph-commons/pull/100/files#diff-1970456b7377f617f05b33f0eee8315fab54943361429d7123f2cda81a98de7aR109
- https://github.com/apache/incubator-hugegraph-commons/pull/100/files#diff-6b23834e19e01fce45a9e2fcf2ce9d7eb9075847f0de9fab1e6c19102f2efc7eR215